node/sc/bridge_manager.go

```diff
@@ -432,20 +433,22 @@ func (bi *BridgeInfo) GetCurrentBlockNumber() (uint64, error) {
 // DecodeRLP decodes the Klaytn
 func (b *BridgeJournal) DecodeRLP(s *rlp.Stream) error {
 	var elem struct {
+		Alias         string
 		LocalAddress  common.Address
 		RemoteAddress common.Address
 		Paired        bool
 	}
 	if err := s.Decode(&elem); err != nil {
```

Comment:
Can this logic decode the legacy journal (without Alias data)?

Reply:
Closed duplicated thread.